rfcs/text/0001-rfc-process.md

```diff
@@ -70,46 +106,124 @@ Where necessary, existing RFCs can be revised using the same process (a proposed
 
 ## Detailed Design
 
+**ADDITION BEGINS**
+
+### Drafting RFCs
+
+#### **Authors**:
+
+There are two types of RFCs that may be proposed: ***informational*** RFCs and ***non informational*** RFCs.
+Non informational RFCs are expected to be linked directly to an existing ticket in the DCP Zenhub board which will 
+usually be tagged as a Spike. Informational RFCs are more speculative and need not directly reference an open ticket in
+the DCP Zenhub board.
+
+Prior to proposing an ***informational*** RFC, it is recommended that authors engage the DCP community to build
```

Comment:
I don't see the need in the case of an Informational RFC. You're just sharing information.

Reply:
Fair enough. I will delete this section